### ISSUE TITLE
Win+E doesn't launch explorer++

### ISSUE THREAD

**Issue by eight04** [derceg/explorerplusplus#439]
**Describe the bug**

After configuring explorer++ to replace the original explorer, you still get the original explorer by pressing Win+E.

**To Reproduce**
1. Configure explorer++ to replace Explorer.
2. Press Win+E.

**Expected behavior**

Launches explorer++

**Screenshots**

**Version (please complete the following information):**
 - Explorer++ version: 1.4.0.2293 (64-bit build)
 - OS: Windows 10

**Additional context**

**Comment by blue-duty** [comment]
I don't think this is a bug,Win + E is a system shortcut, try binding this system shortcut to explorer++

**Comment by eight04** [comment]
> try binding this system shortcut to explorer++

How? Is there a Windows setting for this?

I think Explorer++ should bind it automatically when `Replace Explorer` is checked.

**Comment by blue-duty** [comment]
I used this method: 

> Modify Win + E shortcut keys
Open the registry path: `HKEY_CLASSES_ROOT\Folder\shell\opennewwindow\command` , modify the default value to the program path you want to open, delete the DelegateExecute key (it is recommended to export a copy before deleting it)

**Comment by griffin-warrior** [comment]
In Windows 10, generally, you have to launch the ahk script with administrator privilege so that it can work normally when focusing on privileged windows, such as Task Manager. This fixes my problem of ahk shortcut scripts not always working as expected.

**Comment by stopgaming** [comment]
The registry method you described launches a new instance every time, which feels less intuitive than opening a new tab in the existing window. That's why I made a small helper that sends Ctrl+T to the running Explorer++ instance instead. But honestly, the cleanest solution would be if Explorer++ itself exposed a DDE-like command or a dedicated message (like WM_COMMAND with a known ID) for new tabs. Sending keystrokes indirectly is still a bit of a hack.
